src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -297,6 +297,16 @@ override async void OnActivated(IActivatedEventArgs e)
 			}
 		}
 
+#if !HAS_UNO_WINUI
```

Comment:
I'm not sure why.

Reply:
Same here- the virtual method should not exist in WinUI-based tree